### ISSUE TITLE
NamedChunks default value is not listed in the wiki documentation

### ISSUE THREAD

**Issue by adamdport** [angular/angular-cli#7192]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ X ] feature request
```
Visit https://github.com/angular/angular-cli/wiki/angular-cli. Under defaults->build, every option has a default documented except for namedChunks. It should show the default value (true or false)

**Comment by filipesilva** [comment]
That's a good point. This is because it actually has a variable default. In dev, it is true, but in prod it is false.

It should really be listed in https://github.com/angular/angular-cli/wiki/build#--dev-vs---prod-builds but isn't.

Would you be available to submit a PR for this? This is the file you would have to edit: https://github.com/angular/angular-cli/blob/master/docs/documentation/build.md. An appropriate commit message would be `docs: add NamedChunks to dev/prod defaults table`.

**Comment by filipesilva** [comment]
Fixed by #7253
